src/helics/application_api/MessageFederate.cpp

```diff
@@ -119,6 +119,7 @@ void MessageFederate::registerMessageInterfacesJson (const std::string &jsonStri
         {
             auto eptName = getKey (ept);
             auto type = jsonGetOrDefault (ept, "type", std::string ());
+            auto info = jsonGetOrDefault (ept, "info", std::string ());
```

Comment:
the case that would be problematic for this is if the info was defined in a JSON file as a JSON formatted string.  In that case I am not entirely certain the code would function properly.    I might be able to make it work as such though, but we do need to take that into account that info might be a JSON structure in a JSON file

Reply:
I implemented the changes (once I figured out how to do the modify, the rest of it was fairly trivial and a lot of the garbage I created trying to dig through it could be cleaned up), and you're right, a true JSON string breaks it because of the quotes, however I was able to put an escaped JSON string in without any trouble. I will push up those changes to the branch, and we can evaluate how we want to handle those, without them necessarily interfering with the beta window any further.